experiments/Atmos_LES/bomex.jl

```diff
@@ -0,0 +1,502 @@
+#=
+@article{doi:10.1175/1520-0469(2003)60<1201:ALESIS>2.0.CO;2,
+author = {Siebesma, A. Pier and Bretherton, 
+          Christopher S. and Brown, 
+          Andrew and Chlond, 
+          Andreas and Cuxart, 
+          Joan and Duynkerke, 
+          Peter G. and Jiang, 
+          Hongli and Khairoutdinov, 
+          Marat and Lewellen, 
+          David and Moeng, 
+          Chin-Hoh and Sanchez, 
+          Enrique and Stevens, 
+          Bjorn and Stevens, 
+          David E.},
+title = {A Large Eddy Simulation Intercomparison Study of Shallow Cumulus Convection},
+journal = {Journal of the Atmospheric Sciences},
+volume = {60},
+number = {10},
+pages = {1201-1219},
+year = {2003},
+doi = {10.1175/1520-0469(2003)60<1201:ALESIS>2.0.CO;2},
+URL = {https://journals.ametsoc.org/doi/abs/10.1175/1520-0469%282003%2960%3C1201%3AALESIS%3E2.0.CO%3B2},
+eprint = {https://journals.ametsoc.org/doi/pdf/10.1175/1520-0469%282003%2960%3C1201%3AALESIS%3E2.0.CO%3B2}
+=# 
+
+using Distributions
+using Random
+using StaticArrays
+using Test
+using DocStringExtensions
+using LinearAlgebra
+
+using CLIMA
+using CLIMA.Atmos
+using CLIMA.DGmethods.NumericalFluxes
+using CLIMA.GenericCallbacks
+using CLIMA.Mesh.Filters
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+using CLIMA.VariableTemplates
+
+import CLIMA.DGmethods: vars_state, vars_aux
+import CLIMA.Atmos: source!, atmos_source!, altitude
+import CLIMA.DGmethods: boundary_state!
+import CLIMA.Atmos: atmos_boundary_state!, atmos_boundary_flux_diffusive!, flux_diffusive!, thermo_state
+
+# ---------------------------- Begin Boundary Conditions ----------------- #
+"""
+  BOMEX_BC <: BoundaryCondition
+  Prescribes boundary conditions for Barbados Oceanographic and Meteorological Experiment (BOMEX)
+#Fields
+$(DocStringExtensions.FIELDS)
+"""
+struct BOMEX_BC{FT} <: BoundaryCondition
+  "Friction velocity"
+  u_star::FT  
+  "Sensible Heat Flux"
```

Comment:
Can we change these names to match lines 415-416 as `Potential temperature flux` and `Total moisture flux`?

Reply:
@szy21 @akshaysridhar Should we just move to actual latent heat flux and sensible heat flux here and in the docs? I realize Siebesma et al. had evaporation and potential temperature flux, but the corresponding energy fluxes would be clearer and more intuitive. We can just convert the Siebesma fluxes to energy fluxes using c_p at the surface specific humidity and L_v at surface temperature.

Reply:
I'm in favour of energy fluxes; it's consistent with the prognostic variables being used here. @claresinger this has been modified

Reply:
Sounds good to me. I will modify the docs accordingly.

Reply:
Let's use SHF = 9.5 and LHF = 147.2. I have updated the docs.

Reply:
Fixed.